README.md

```diff
@@ -1,2 +1,146 @@
-# leaderboard
-A leaderboard connected to an API to display the top scores of a game.
+<div align="center">
+
+  <h1><b>Leaderboard</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
```

Comment:
- Hey, since you won't be adding a test to this project, please remove this section from the readme to make your readme have the correct content relevant to the project.

Reply:
Updated, thanks!